apps/addons/cron.py

```diff
@@ -453,9 +453,7 @@ def reindex_addons(index=None, addon_type=None):
     from . import tasks
     ids = (Addon.objects.values_list('id', flat=True)
            .filter(_current_version__isnull=False,
-                   status__in=amo.VALID_STATUSES,
-                   is_listed=True,
-                   disabled_by_user=False))
```

Comment:
I don't think it was faulty before. It wasn't really necessary to index these previously. But now we have bad data in the index for these add-ons, so we need to do at least one run that doesn't exclude them.